tests/integration/tests.mk

```diff
@@ -41,41 +41,61 @@ else
+	--istio.test.select -multicluster \
+	--istio.test.env kube \
+	${_INTEGRATION_TEST_FLAGS} \
+	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
+
+# Generate integration test targets for multicluster kubernetes environment.
+test.integration.%.kube: | $(JUNIT_REPORT)
+	$(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
+	--istio.test.select +multicluster \
 	--istio.test.env kube \
 	${_INTEGRATION_TEST_FLAGS} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
 # filter out non-standard test directories
 TEST_PACKAGES = $(shell go list ./tests/integration/... | grep -v /qualification | grep -v /examples)
 
-# Generate integration test targets for local environment.
+# Generate integration test targets for local environment. Only single-cluster tests are supported for local env.
 test.integration.%.local: | $(JUNIT_REPORT)
 	$(GO) test -p 1 ${T} -race ./tests/integration/$(subst .,/,$*)/... \
+	--istio.test.select -multicluster \
 	--istio.test.env native \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
-# Generate presubmit integration test targets for each component in kubernetes environment
+# Generate presubmit integration test targets for each component in single-cluster kubernetes environment
 test.integration.%.kube.presubmit: istioctl | $(JUNIT_REPORT)
 	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
-	--istio.test.select -postsubmit,-flaky \
+	--istio.test.select -postsubmit,-flaky,-multicluster \
+	--istio.test.env kube \
+	${_INTEGRATION_TEST_FLAGS} \
+	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
+
+# Generate presubmit integration test targets for each component in multi-cluster kubernetes environment
+test.integration.%.kube.multicluster.presubmit: istioctl | $(JUNIT_REPORT)
+	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
+	--istio.test.select -postsubmit,-flaky,+multicluster \
 	--istio.test.env kube \
 	${_INTEGRATION_TEST_FLAGS} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
 # Generate presubmit integration test targets for each component in local environment.
+# Only single-cluster tests are supported for local env.
 test.integration.%.local.presubmit: | $(JUNIT_REPORT)
 	$(GO) test -p 1 ${T} -race ./tests/integration/$(subst .,/,$*)/... \
-	--istio.test.env native --istio.test.select -postsubmit,-flaky \
+	--istio.test.select -postsubmit,-flaky,-multicluster \
+	--istio.test.env native \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
-# Presubmit integration tests targeting Kubernetes environment.
+# Presubmit integration tests targeting single-cluster Kubernetes environment.
 .PHONY: test.integration.kube.presubmit
 test.integration.kube.presubmit: istioctl | $(JUNIT_REPORT)
 	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ${TEST_PACKAGES} -timeout 30m \
-    --istio.test.select -postsubmit,-flaky \
+    --istio.test.select -postsubmit,-flaky,-multicluster \
```

Comment:
maybe outside of the scope of this, but the labels should really be inverted I think. If I run `go test ./...` it would be nice if things just worked. All of the non standard things, like multicluster, would be opt-in, not opt out.

Not a big deal though

Reply:
Seems reasonable, but agree it's outside of the scope here. Mind capturing it in an issue?